.github/workflows/TerraformTest-cloud-function.yml

```diff
@@ -0,0 +1,49 @@
+name: Terraform Test - wfl-instance module
```

Comment:
We might need to be a little careful here as the behavior of `fmt` can vary a bit between tf versions. I definitely like running these lints in ci.

Reply:
This is true--I suppose even if the call-site has newer TF, it won't try to reformat this at all, so it would only be a problem when we are actively messing with the source code here.

Reply:
yup I think this is fine for the record, just wanted to call attention a potential when someone may have `fmt` auto run no save with a different version